spec/factories/responses.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :response do
+    survey_response nil
```

Comment:
I think it might be auto-generated to nil so tests don't fail even if you haven't declared the factories for the associated tables. If you remove the nil (i.e. just `survey_response`), FactoryGirl should be able to find and create an instance of the other records. This makes it nice when we start writing more complex tests for controllers where we may have to call `response.survey_response` and the `survey_response` record has already been created.